main/src/main/java/org/mobilitydata/gtfsvalidator/validator/AgencyConsistencyValidator.java

```diff
@@ -94,4 +110,16 @@ public void validate(NoticeContainer noticeContainer) {
       }
     }
   }
+
+  /**
+   * Checks is an `agency.agency_id` is missing or blank i.e. `agency_id` is a sequence of
+   * whitespaces
+   *
+   * @param agency the {@code GtfsAgency} to check the id
+   * @return true if `agency.agency_id` is missing or blank (i.e. `agency_id` is a sequence of
+   *     whitespaces), false otherwise
+   */
+  private boolean isAgencyIdMissingOrBlank(GtfsAgency agency) {
+    return !agency.hasAgencyId() || StringUtils.isBlank(agency.agencyId());
```

Comment:
`StringUtils.isBlank(agency.agencyId())` will be always false. I am leaving to you as an exercise to figure out why this is so.

Reply:
https://github.com/MobilityData/gtfs-validator/pull/799#issuecomment-799704893